homeassistant/components/airnow/strings.json

```
@@ -25,6 +25,9 @@
     "sensor": {
       "o3": {
         "name": "[%key:component::sensor::entity_component::ozone::name%]"
+      },
+      "station": {
+        "name": "PM2.5 reporting station"
       }
```

```suggestion
        "name": "PM2.5 reporting station",
        "state_attributes": {
          "lat": {
            "name": "Latitude"
          },
          "long": {
            "name": "Longitude"
          }
        }
      }
```
Can you try this and then run `python3 -m script.translations develop --all` and retest?

Note: I explicitly didn't use common config flow translations as that would be a different context of strings

---

Thanks, works now